<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sequence_build</class>
 <widget class="QWidget" name="Sequence_build">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sequence - FILE</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="3,0">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,0,0">
     <item>
      <widget class="QListView" name="listSequence">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragDrop</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::MoveAction</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="textElideMode">
        <enum>Qt::ElideRight</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeWidget" name="treeOptions">
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragOnly</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <column>
        <property name="text">
         <string>1</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>Individual setpoints</string>
        </property>
        <property name="flags">
         <set>ItemIsUserCheckable|ItemIsEnabled</set>
        </property>
        <item>
         <property name="text">
          <string>Set Temperature</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled</set>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Set Field</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled</set>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Wait</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled</set>
         </property>
        </item>
       </item>
       <item>
        <property name="text">
         <string>Measurements</string>
        </property>
        <property name="flags">
         <set>ItemIsUserCheckable|ItemIsEnabled</set>
        </property>
        <item>
         <property name="text">
          <string>Resistivity vs Temperature</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled</set>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Resistivity vs Field</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsEnabled</set>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hall-resistivity vs Temperature</string>
         </property>
         <property name="flags">
          <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsUserCheckable</set>
         </property>
        </item>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="lineFileLocation"/>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="pushSaving">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
